nums/numpy/api.py

```diff
@@ -330,6 +330,9 @@ def squeeze(a: BlockArray, axis=None):
     assert axis is None, "axis not supported."
     return a.squeeze()
 
+def swapaxes(x: BlockArray, axis1: int, axis2: int):
```

Comment:
Can we change the name of the input array to be `a` instead of `x`? This is the variable name in numpy, in case keyword arguments are used.

Reply:
I've changed the name